test/JDBC/input/BABEL-1206-vu-verify.sql

```diff
@@ -0,0 +1,57 @@
+select (case when a<b then 1 else 0 end) r from babel_1206_vu_prepare_t1;
+go
+select (case when a<=b then 1 else 0 end) r from babel_1206_vu_prepare_t1;
+go
+select (case when a=b then 1 else 0 end) r from babel_1206_vu_prepare_t1;
+go
+select (case when a>b then 1 else 0 end) r from babel_1206_vu_prepare_t1;
+go
+select (case when a>=b then 1 else 0 end) r from babel_1206_vu_prepare_t1;
+go
+
+
+CREATE TABLE babel_1206_vu_verify_t1(
```

Comment:
why are we doing these in verify instead of prepare?

Reply:
Resolved in latest commit